pyproject.toml

```diff
@@ -31,7 +31,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```

Comment:
This will be addressed shortly, likely was from a copy/paste from other project https://github.com/networktocode/netutils/pull/75